package.json

```diff
@@ -57,4 +59,4 @@
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0"
   }
-}
+}
```

Comment:
- [ ] Great job so far howver you should use the  `createAsyncThunk` in the project for creating thunk action creators. Kindly add this. Your code might looks like:

```js

import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
import axios from 'axios'; 


export const getBooks = createAsyncThunk(
  'books/getBooks',
  async () => {
    const response = await axios.get(
      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/',
    );
    return response.data;
  },
); 
```

Reply:
In my books.js I have used the createAsyncThunk. Do you suggest that I have them configured wrongly? kindly elaborate

Reply:
Hey @brainskev, I could not see it. let's me check again!!